docs/fastdds/ros2/discovery_server/ros2_discovery_server.rst

```diff
@@ -311,33 +310,37 @@ ROS 2 Command Line Interface (CLI) implements several introspection features to
 execution.
 These features (i.e. `rosbag`, `topic list`, etc.) are very helpful to understand a ROS 2 working network.
 
-Most of these features use the DDS capability to share any topic info with every exiting participant.
+Most of these features use the DDS capability to share any topic information with every exiting participant.
 However, the new :ref:`ros2_discovery_server_v2` implements a traffic network reduction
 that limits the discovery data between nodes that do not share a topic.
 This means that not every node will receive every topic data unless it has a reader in that topic.
+As most of ROS 2 CLI Introspection is executed by adding a node into the network (some of them use ROS 2 Daemon,
+and some creates their own nodes), using Discovery Server v2 we will find that most of these functionalities are
+limited and do not have the whole information.
 
-The Discovery Server v2 functionality allows every server to know and share all the participants and topic information
-with every other server matched.
-So, this functionality can be used in order to use ROS 2 introspection successfully when running with Discovery Server
-discovery protocol.
+The Discovery Server v2 functionality allows every node running as a **Server** (Participant/Node type)
```

Comment:
Is the parenthesis intended to be a reference to the API?

Reply:
No. It intended to be a definition of `Server` as a Participant type / Node type.
I will leave just participant so it makes it more comprehensible